src/Propel/Generator/Manager/AbstractManager.php

```diff
@@ -374,7 +374,7 @@ protected function includeExternalSchemas(\DOMDocument $dom, $srcDir)
             // The external schema may have external schemas of its own ; recurs
             $this->includeExternalSchemas($externalSchemaDom, $srcDir);
             foreach ($externalSchemaDom->getElementsByTagName('table') as $tableNode) {
-                if ($referenceOnly) {
+                if ($referenceOnly == "true") {
```

Comment:
Not critical but it'll be better to use constant values in left side of comparison expression.